src/CONST/index.ts

```diff
@@ -1006,6 +1006,7 @@ const CONST = {
         WALLET_AGREEMENT_URL: `${EXPENSIFY_URL}/expensify-payments-wallet-terms-of-service`,
         BANCORP_WALLET_AGREEMENT_URL: `${EXPENSIFY_URL}/bancorp-bank-wallet-terms-of-service`,
         EXPENSIFY_APPROVED_PROGRAM_URL: `${USE_EXPENSIFY_URL}/accountants-program`,
+        FEES_URL: `${EXPENSIFY_URL}/fees`,
```

Comment:
You don't need this because it's already defined at the root, and nothing uses `CONST.OLD_DOT_PUBLIC_URLS.FEES_URL`?

```suggestion
```

Reply:
+1